src/ArticleCard.jsx

```diff
@@ -15,6 +16,23 @@ function ArticleCard() {
       setIsLoading(false);
     });
   }, [article_id]);
+
+  useEffect(() => {
+    patchVotes(article_id).then((votes) => {
+      setVoteCount(votes);
+    });
+  }, [article_id]);
+
+  const increment = () => {
```

Comment:
The increment and decrement functions need to have the patchVotes functionality inside them. Be careful though - one should increase and one should decrease the votes in the database, but at the moment it only increases